src/test/regress/expected/multi_sequence_default.out

```diff
@@ -738,8 +742,12 @@ SELECT nextval('seq_12');
 -- nextval from worker should fail for int and smallint sequences
 SELECT nextval('seq_13');
 ERROR:  nextval: reached maximum value of sequence "seq_13" (2147483647)
+DETAIL:  nextval(sequence) calls in worker nodes are not supported for column defaults of type int or smallint
+HINT:  If the command was issued on a distributed table / sequence from a worker node, try issuing it from the coordinator node instead.
 SELECT nextval('seq_14');
 ERROR:  nextval: reached maximum value of sequence "seq_14" (32767)
+DETAIL:  nextval(sequence) calls in worker nodes are not supported for column defaults of type int or smallint
+HINT:  If the command was issued on a distributed table / sequence from a worker node, try issuing it from the coordinator node instead.
```

Comment:
Maybe `If the command was issued on a distributed table / sequence from a worker node` -> `If the command was issued from a worker node` ?

sounds less messy, and there are a lot of varieties that are not covered (e.g. reference tables)